### ISSUE TITLE
Wrong caret positioning when Enter on empty block

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/gutenberg-mobile#848]
After inserting a new block by tapping ENTER at the end of another Paragraph block, then tapping ENTER again will append another block and place the caret at the end of the first block.

![newblockcaret](https://user-images.githubusercontent.com/6597771/55812202-ed1f8f80-5ac0-11e9-8971-169b283570a8.gif)

Steps to reproduce
1. write something and tap ENTER to create a new block
2. on the new block tap ENTER again
3. observe a new block is appended, and the caret goes back to the previous block.

**Comment by mkevins** [comment]
I could not reproduce this, but had a similar issue that was just fixed with https://github.com/wordpress-mobile/gutenberg-mobile/pull/857 . @mzorz I wonder if that PR also fixes this issue?

**Comment by mzorz** [comment]
> I could not reproduce this, but had a similar issue that was just fixed with #857 . @mzorz I wonder if that PR also fixes this issue?

That's true @mkevins ! just tested and the previously observed behavior doesn't seem to happen anymore.

Closing via #857

**Comment by mzorz** [comment]
> I could not reproduce this, but had a similar issue that was just fixed with #857 . @mzorz I wonder if that PR also fixes this issue?

That's true @mkevins ! just tested and the previously observed behavior doesn't seem to happen anymore.

Closing via #857
